<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Books</class>
 <widget class="QWidget" name="Books">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_bookImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>191</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>191</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>161</width>
     <height>191</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_bookInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lb_area">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_connectBoss">
      <property name="text">
       <string>联系卖家</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>价格：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_Price">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
